app.js

```diff
@@ -22,13 +23,34 @@ var http        = require('http'),
 // Read in environment variables
 dotenv.config({path: '.env.local'});
 if (process.env.NODE_ENV === 'production') {
-  dotenv.config({path: '/etc/ood/config/apps/files/env'});
+    dotenv.config({path: '/etc/ood/config/apps/files/env'});
 }
 
+var startsWithAny = function(subject, prefixes){
+    return prefixes.some(function(x){
+        return subject.startsWith(x);
+    });
+};
+
+var whitelist = {
+    paths:  process.env.WHITELIST ? process.env.WHITELIST.split(":") : [],
+    enabled: function(){ return this.paths.length > 0; },
+    contains: function(filepath){
+        return this.paths.filter(function(whitelisted_path){
```

Comment:
This method should call `path.realpath(filepath)` (and handle the exception case etc.) to avoid filepath being a symlink, prior to testing against whitelist